nubis/puppet/consul.pp

```diff
@@ -36,11 +36,9 @@
     group            => 'root',
 }
 
-package { "tar":
```

Comment:
Actually, no, it fixes the problem!

The issue is that some other puppet module, consul_template, IIRC, already includes package { "tar", and this
was causing a package redefined error.

So, this is the correct fix, actually, even if I recognize it's not obvious.